### DISCUSSION TITLE
Using SeleniumBase Live?

### DISCUSSION THREAD

**Discussion by Meathelix1** [seleniumbase/SeleniumBase#2846] [Q&A] [answered]
I have a question, so I want to connect to my browser remotely using SeleniumBase, however I want to be able to interact with the pages live, not just test but instead a live interaction. 

Could this be done?

Thanks

**Comment by mdmintz** [comment, marked as answer]
Sounds like a duplicate of https://github.com/seleniumbase/SeleniumBase/issues/2049 (Chrome Remote Debugging)

Unless you are referring to something like a Python `breakpoint()`, which lets you pause the automation so that you can run your own commands from the command-line.

**Reply by Meathelix1** [reply]
I was able to get it to pause without breakpoint and enabling Chrome Remote Debugging.

Basically what I have done is I have built an API connecting from NextJS to SeleniumBase and if a user inputs text or interacts with the NextJS Application it will boot up a SeleniumBase Instance and they can interact with the certain website from the NextJS App. 

I prob could of done it a better way but SeleniumBase is still the king for not being detected as a bot on the web.

Love the quick response.
